subsys/usb/class/bluetooth.c

```diff
@@ -33,9 +33,11 @@ static K_FIFO_DEFINE(tx_queue);
 #define BLUETOOTH_IN_EP_ADDR		0x82
 
 /* TODO: Replace use of USB_MAX_FS_INT_MPS if higher speeds are supported */
-#define BLUETOOTH_BULK_EP_MPS           MIN(BT_BUF_ACL_SIZE, \
-					    USB_MAX_FS_BULK_MPS)
-#define BLUETOOTH_INT_EP_MPS            MIN(BT_BUF_RX_SIZE, USB_MAX_FS_INT_MPS)
+/* Some usb drivers only accept EP sizes that are a power of 2 */
+#define BLUETOOTH_BULK_EP_MPS           ROUND_UP(MIN(BT_BUF_ACL_SIZE, \
```

Comment:
The valid sizes for FS bulk endpoint are 8, 16, 32, or 64 bytes, but maybe we should just set it to USB_MAX_FS_BULK_MPS?

Reply:
yes this seems sensible enough, I'll do this and push again

Reply:
`ROUND_UP` does not rounds to the next power of 2, it rounds to the next multiple of 2. If `BT_BUF_ACL_SIZE=9` the
`BLUETOOTH_BULK_EP_MPS` would get value of 10.

Reply:
Yes, my mistake. I did this because the value was 31 in my config, so ROUND_UP effectively rounded up to 32. Now I updated it to always use the max bulk size (i.e. 64). Actually I don't know if I have to do this for the INT_EP, do you know if the spec allow arbitrary sizes ?

Reply:
"The maximum allowable interrupt data payload size is 64 bytes or less for full-speed" from USB 2.0 spec

Reply:
okay, I can revert the changes to this EP then